### DISCUSSION TITLE
Maybe the build directory should be added to .gitignore file?

### DISCUSSION THREAD

**Discussion by gogim1** [apache/kvrocks#631] [General]
According to [README.md](https://github.com/apache/incubator-kvrocks/blob/unstable/README.md#building-kvrocks), there will be lots of untracked files in the `build` directory after ` ./build.sh build `

**Comment by git-hulk** [comment]
Yes, would you like to submit a PR to resolve it?

**Reply by gogim1** [reply]
Done. Please take a look when you are free. Thanks. 😃
#632
